### PR TITLE
separate the init config in helm

### PR DESCRIPTION
**Pull request by stan-dot**
long branch name due to @DiamondJoseph choice of words in the issue name, but at least it's entertaining

### CODE REVIEW

File: helm/blueapi/values.yaml [hunk 1]
```diff
@@ -108,12 +108,13 @@ worker:
...
-  #   repositories: []
-  #     - name: "dodal"
-  #       remote_url: https://github.com/DiamondLightSource/dodal.git
+
+initContainer:
+  scratch:
+   root: /blueapi-plugins/scratch
+    repositories: []
+      - name: "dodal"
+        remote_url: https://github.com/DiamondLightSource/dodal.git
```
Comment (DiamondJoseph) [inline helm/blueapi/values.yaml:117]: This yaml is invalid for both whitespace and setting a value twice. repositories should be `[]` with a commented out example for dodal. Does an auto format touch this file?

File: helm/blueapi/templates/statefulset.yaml [hunk 2]
```diff
@@ -59,8 +64,9 @@ spec:
             if [ $? -ne 0 ]; then echo 'Blueapi failed'; exit 1; fi;
             echo "Exporting venv as artefact"
             cp -r /venv/* /artefacts
+
```
Comment (DiamondJoseph) [inline helm/blueapi/templates/statefulset.yaml:67]: remove whitespace

**Review by DiamondJoseph — CHANGES_REQUESTED** [review]
Looks conceptually correct, some nits.

**Review by callumforrester — COMMENTED** [review]
Concept looks good, will test it when the PR is marked as ready

File: helm/blueapi/templates/statefulset.yaml (hunk 2 above)
Comment (stan-dot) [reply helm/blueapi/templates/statefulset.yaml]: done

**Review by callumforrester — APPROVED** [review]

**Review by DiamondJoseph — CHANGES_REQUESTED** [review]
1 nit then LGTM

File: helm/blueapi/values.yaml [hunk 3]
```diff
@@ -80,7 +80,8 @@ listener:
   resources: {}
 
 # Additional envVars to mount to the pod as a String
-extraEnvVars: []
+extraEnvVars:
+  []
```
[the remark this answers is not in the document]
Comment (stan-dot) [reply helm/blueapi/values.yaml:84]: my yaml parser corrects to the way it was,

**Pull request merged**